test/16_recode.jl

```diff
@@ -29,6 +29,42 @@ const ≅ = isequal
     end
 end
 
+@testset "recode_in" begin
+    @testset "collection is a string" begin
+        @test CategoricalArrays.recode_in("a", "b") == false
+        @test CategoricalArrays.recode_in(missing, "b") == false
+    end
+    @testset "collection without missing" begin
+        @test CategoricalArrays.recode_in(1, [1, 2]) == true
+        @test CategoricalArrays.recode_in(1, [2, 3]) == false
+    end
+    @testset "collection with missing" begin
+        @test CategoricalArrays.recode_in(1, [1, 2, missing]) == true
+        @test CategoricalArrays.recode_in(1, [2, missing]) == false
+        @test CategoricalArrays.recode_in(missing, [1, 2, missing]) == true
+    end
+    @testset "collection is a single value" begin
+        @test CategoricalArrays.recode_in(1, 1) == true
+        @test CategoricalArrays.recode_in(1, missing) == false
+        @test CategoricalArrays.recode_in(missing, missing) == false
+    end
+end
```

Comment:
Can you also try nested collections, like an array `in` an array of arrays? Also testing tuples would be nice.